### DISCUSSION TITLE
Presto hive connector

### DISCUSSION THREAD

**Discussion by sparkheo7** [prestodb/presto#15689] [Q&A]
Does presto support hive connector on Hadoop 3.X ?

**Comment by bitsondatadev** [comment]
@sparkheo7 Presto does not support Hadoop 3.X.

Trino (formerly known os PrestoSQL) is a similar but more feature rich query engine that [branched off of Presto about 2 years ago by the original founders of Presto](https://trino.io/blog/2020/12/27/announcing-trino.html) and it is able to read, insert, delete, and we recently just [added update support](https://github.com/trinodb/trino/pull/5861)! You can see the entire [list of HDP3 compatibility](https://github.com/trinodb/trino/issues/1218) that we are working through. 

Here are a few other resources around HDP3 support. Let me know if you have any questions!
 - [Interview with David Stryker](https://trino.io/episodes/4.html) (the engineer behind a lot of the HDP3 changes).
 - Blog on [ACID support](https://trino.io/blog/2020/06/01/hive-acid.html) in Trino.

**Reply by sparkheo7** [reply]
This is helpful. Let me explore Trino. 
Thank you for your quick reply.
